src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    //declarations
+    public static String line = "------------------------------------------------------------------------------------------\n";
+
+    //declare task array and keep track of how many tasks stored
+    public static Task[] t = new Task[100];
```

Comment:
No coding violations found, but maybe you can declare 100 as a constant to indicate that it is the maximum size of the list.